chidioguejiofor/maintenanceApp
# #157994493: user should be able to create, update and view requests in the app

### What does this PR do?
- syncs update page to consume API
- improves the styling of requests page
- replaced request details page with modal
- improved UX in create and update request pages
- implements feedback for signup form

### Description of task to be completed
- Consume API for users functions

### Any background context you want to provide?
- Updates heroku app

### How can this be tested

- Sign up to the app via https://chidiebere-maintenance.herokuapp.com/UI/html/signinSignup.html?action=signup
![correctvalues](https://user-images.githubusercontent.com/38565349/41189305-1e7d16ea-6bcc-11e8-8f7b-0443577e060a.PNG)
- You would see the updated client page
![new client page](https://user-images.githubusercontent.com/38565349/41189328-7eaf325a-6bcc-11e8-8754-7662d8bdec63.PNG)

- Clicking on the requests page show a modal with the details of the request like so
![requesst details](https://user-images.githubusercontent.com/38565349/41189383-421fa9fe-6bcd-11e8-957a-74eff6903f5d.PNG)
- You can navigate the through the app and see the create and update pages

### What are the relevant Github Issues?
- None

### Credits
- The idea of using modals to display request details was inspired by w3school document on modals
